3rdparty/jvm/com/github/scopt/BUILD

```diff
@@ -0,0 +1,11 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# DOCSTART: This target is included in documentation.
```

Comment:
I don't think that this is true? Maybe copied from somewhere...

Regarding `scopt`: fine with using it for this. But consider adding a comment in `src/scala/org/pantsbuild/zinc/options/OptionSet.scala` recommending using `scopt` instead?